index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
```

Comment:
Kindly add the 'viewport' meta element which was not specified. Fix Linter error.